### PR TITLE
Add a Link component

### PR DESCRIPTION
**Pull request by wfro**
Adds a themer-aware `<Link />` component for hyperlinks.

### CODE REVIEW

File: src/components/Link/docs/Link.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Links!
```
Comment (dcocchia) [inline src/components/Link/docs/Link.md]: Might want a tad more about why one may need/want a Link component :D

Comment (wfro) [reply src/components/Link/docs/Link.md]: LOL yes, forgot to fill in this section.

**Comment by wfro** [comment]
@dcocchia pushed another commit with a better docs description than "LInks!" which nullified the last review. I think this is ready to go though. We can possibly tweak the API based on how it gets used.

File: src/components/Link/__tests__/Link.spec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import Link from '../Link'
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { StyleRoot } from 'radium'
+import { mount } from 'enzyme'
+import { spy, match } from 'sinon'
+
+fdescribe('Link', () => {
```
Comment (dcocchia) [inline src/components/Link/__tests__/Link.spec.js]: This looks like a typo, should this be `describe`?

File: src/components/Link/docs/Link.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+The `Link` component is valuable mainly because it's hooked up to Themer.
```
Comment (dcocchia) [inline src/components/Link/docs/Link.md]: if this is the main value prop of this component, we should add a test or two to test that themer changes change the component output

File: src/components/Link/__tests__/Link.spec.js (hunk 2 above)
Comment (wfro) [reply src/components/Link/__tests__/Link.spec.js]: Ah yeah, this is a test I forgot to un-focus.

**Pull request merged**